client/my-sites/plans-v2/plans-filter-bar/index.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+
+import React, { useState } from 'react';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import SegmentedControl from 'components/segmented-control';
+import SelectDropdown from 'components/select-dropdown';
+import { TERM_MONTHLY, TERM_ANNUALLY } from 'lib/plans/constants';
+import type { Duration, ProductType } from '../types';
+import { ALL, PERFORMANCE, SECURITY } from '../constants';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const productTypeOptions = {
+	[ SECURITY ]: {
+		id: SECURITY,
+		label: translate( 'Security' ),
+	},
+	[ PERFORMANCE ]: {
+		id: PERFORMANCE,
+		label: translate( 'Performance' ),
+	},
+	[ ALL ]: {
+		id: ALL,
+		label: translate( 'All' ),
+	},
+};
```

Comment:
Maybe should be in the `constants` file?

Reply:
Good idea.